posts/2020-12-15-rust-survey-2020.md

```diff
@@ -0,0 +1,118 @@
+
+### C++ Interop
+
+Interestingly, C++ interop was the most requested language for better interop with Rust with C and Python in second and third place. Improved C++ interop was especially often mentioned as a way to improve Rust usage specifically at work. In fact, for users who work on large codebases (100,000 lines of code or larger) C++ interop and unsurprisingly compile times were the most cited ways to improve their Rust experience.    
+
+![If you want better language interop, with which language?](/images/2020-12-rust-survey-2020/language-interop.svg)
+
+### Improved Learnability
+
+When asked how to improve adoption of Rust, many cited making Rust easier to learn with 15.8% of respondents saying they would use Rust more if it were “less intimidating, easier to learn, or less complicated”. Additionally when directly asked how people think we can improve adoption of Rust, the largest category of feedback was documentation and training. 
+
+When we asked to rate their expertise in Rust, there was a clear peak at 7 out of 10. It’s hard to say how this compares across languages but it seems notable that relatively few are willing to claim full expertise. However, when compared with last year, the Rust community does seem to be gaining expertise in the language. 
+
+![How would you rate your expertise in Rust?](/images/2020-12-rust-survey-2020/rust-expertise-overall.svg)
+
+We also asked about the difficulty of specific topics. The most difficult topic to learn according to survey results is somewhat unsurprisingly lifetime management with 61.4% of respondents saying that the use of lifetimes is either tricky or very difficult.  
+
+![Percent of respondents rating each topic as tricky or very difficult.](/images/2020-12-rust-survey-2020/topic-difficulty-ratings.svg)
+
+It does seem that having C++ knowledge helps with 20.2% of respondents with at least some C++ experience noting lifetimes to be “very difficult” while 22.2% of those without C++ knowledge found the topic to be “very difficult”. Overall, systems programming knowledge (defined as at least some experience in C and C++), tends to make for more confident Rust users: those with systems programming experience rating themselves as 5.5 out of 10 on their Rust expertise, while those with experience in statically typed garbage collected languages like Java or C# rating themselves as 4.9 out of 10. Those with only experience in dynamically typed languages like Ruby or JavaScript, rated themselves as 4.8 out of 10. 
+
+Unsurprisingly, the more often people use Rust, the more they feel they are experts in the language with 56.3% of those who use Rust daily ranking themselves as 7 or more out of 10 on how much of an expert they are on Rust compared with 22% of those who use Rust monthly. 
+
+![How would you rate your expertise in Rust? (Daily Rust users)](/images/2020-12-rust-survey-2020/rust-expertise-daily.svg)
+
+### Better Support
```

Comment:
This heading is very vague. Maybe 

### Compilation time, library ecosystem

Reply:
How about this?

### Improving Rust

Reply:
That's already the topic up above, right? I think what I would actually do is to break this section into multiple, smaller sections. e.g., one about compilation time and one about library support, and then give those sections more targeted names (my goal is that somebody browsing the headings gets a pretty good idea of the content of the post)

Reply:
@nikomatsakis pushed a change. Let me know what you think!

Reply:
Looks great!